Tests/test_Muscle_tool.py

```diff
@@ -103,10 +107,13 @@ def test_Muscle_with_options(self):
         # Use property:
         cmdline.objscore = "sp"
         cmdline.noanchors = True
-        self.assertEqual(str(cmdline), _escape_filename(muscle_exe) +
-                         " -in Fasta/f002" +
-                         " -out Fasta/temp_align_out2.fa" +
-                         " -objscore sp -noanchors")
+        self.assertEqual(
+            str(cmdline),
+            _escape_filename(muscle_exe)
+            + " -in Fasta/f002"
+            + " -out Fasta/temp_align_out2.fa"
+            + " -objscore sp -noanchors",
+        )
```

Comment:
Those three strings can be merged, might just fit on one line?

Reply:
Done.